src/appkit.cs

```diff
@@ -1232,6 +1260,16 @@ interface NSBezierPath : NSCoding, NSCopying {
 
 		[Export ("flatness")]
 		nfloat Flatness { get; set; }
+
+		[Mac (10,13)]
+		[Export ("appendBezierPathWithCGGlyph:inFont:")]
+		void AppendBezierPathWithCGGlyph (CGGlyph glyph, NSFont font);
```

Comment:
Added the simple overloads. I didn't bother converting them all to appends with an enum (https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#different-objective-c-selectors-who-lead-to-identical-c-signatures) because they are both old bindings and I think that's uglier than what we already have.